### ISSUE TITLE
Make SSP player accessible to screen readers

### ISSUE THREAD

**Issue by itmaybejj** [CastosHQ/Seriously-Simple-Podcasting#312]
The SSP player controls are currently on \<div\> and \<a\> elements, without accessible labels. This means keyboard controls can't focus them (pressing the tab key jumps right past them) and screen readers can't "see" them, either to announce their function or to let the screen reader user interact with them. See https://www.w3.org/TR/WCAG21/#keyboard-accessible and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Needed changes (the easy way, for which I'm about to submit a pull request):

1. each button that is a div or span needs an attribute: role='button'
2. each button that does not contain text needs a label: aria-label='What I will do when you click me'
3. each button that is a div or span needs to be made focusable: tabindex='0'
4. a JS handler needs to be added to simulate a click when the Enter/Return key or spacebar is pressed while the element is focused. (Note that in my code I've just duplicated your handlers...this is crude but abstracting all the handlers into separate functions was more than I'm up for volunteering for).

Alternate way to do it (the hard way, which probably isn't worth the hassle):
1. rewrite every interactive element to use a \<button\> tag

I've also for good measure made the album art clickable on the SSP player. This is not an accessibility thing; rather a usability thing -- when I passed a beta of a site around, everybody tried to click it. Since this button is redundant, and the image is missing an alt, I set it to aria-hidden='true' to hide it from screen readers altogether.
